composer.json

```diff
@@ -35,14 +35,25 @@
 						"url": "https://meta.svn.wordpress.org/sites/",
 						"reference": "trunk/wordpress.org/public_html/wp-content/themes/pub/wporg/"
 					}
+				},
+				{
+					"name": "wordpress-meta/pub",
```

Comment:
I'm getting `wordpress-meta/pub could not be downloaded` when running `composer install`, any ideas? I'm not sandboxed, and my proxy seems fine.

Reply:
Huh, weird. I just tried deleting the `pub` folder and re-running `composer install` and it's still working for me. You should even need the proxy anyway since it's meta.svn.

I copied this over from Learn's composer file because we use the locale stuff there too. If you run `composer install` on your Learn checkout do you get the same error?

Reply:
yeah, same thing on learn, `wordpress-meta/locale-detection could not be downloaded` - the interesting thing is that it claims that `svn: command not found`, but I definitely have SVN here.

In any case, I did a few manual steps and got it working enough to test.

@StevenDufresne Did you run `yarn run create` on this branch, without issues? If it's just my machine, I can figure it out later, but if you have trouble too, we should figure that out.

Reply:
[OS X recently removed it's bundled `svn`](https://stackoverflow.com/questions/60869347/mac-command-line-tools-11-4-no-longer-has-svn), so if you also have it installed via homebrew, then it could be that composer is using the wrong path or something.

I ran into something like that w/ git-svn, and had to [muck w/ paths to fix it](https://github.com/Homebrew/homebrew-core/issues/52490#issuecomment-792604853)